creativecloud/blocks/interactive-marquee/interactive-marquee.js

```diff
@@ -12,6 +12,8 @@ function decorateText(el, createTag) {
     const bodyEl = headingEl.nextElementSibling;
     bodyEl?.classList.add(`body-${typeSize[1]}`);
     bodyEl?.nextElementSibling?.classList.add(`body-${typeSize[1]}`, 'pricing');
+    const bodyElClone = bodyEl?.cloneNode(true);
+    if (bodyEl) { el.insertAdjacentElement('afterEnd', bodyElClone); }
```

Comment:
@sharath-kannan If I author the pricing in this marquee the text is copied twice and pricing is removed
https://mwpw-145612--cc--sharath-kannan.hlx.page/drafts/mathuria/i-marquee
One way to do this would be to individually pick only required content (icon, title, detail and action-area) as there are many other components that would get impacted if content is moved out of the text.